go/test/endtoend/cluster/cluster_process.go

```diff
@@ -679,7 +679,10 @@ func NewBareCluster(cell string, hostname string) *LocalProcessCluster {
 		// path/to/whatever exists
 		cluster.ReusingVTDATAROOT = true
 	} else {
-		_ = createDirectory(cluster.CurrentVTDATAROOT, 0700)
+		err = createDirectory(cluster.CurrentVTDATAROOT, 0700)
+		if err != nil {
+			log.Infof("err in creating file : %s", err.Error())
```

Comment:
In general you should use the error log for errors. 

Why/when should this NOT be fatal?